external/microprofile/CMakeLists.txt

```diff
@@ -25,4 +25,12 @@ openrw_target_apply_options(
     TARGET microprofile
 )
 
+set_target_properties(microprofile
+    PROPERTIES
+        CXX_EXTENSIONS OFF
+        CXX_STANDARD 11
+        C_EXTENSIONS OFF
+        C_STANDARD 99
```

Comment:
also

Reply:
I was trying to be conservative for microprofiler.

Reply:
And it does not matter. Since microprofiler is a c++ only project.